token/cli/src/output.rs

```diff
@@ -257,6 +259,20 @@ impl fmt::Display for CliTokenAccount {
             writeln!(f, "* Please run `spl-token gc` to clean up Aux accounts")?;
         }
 
+        if self.has_permanent_delegate {
+            writeln!(f)?;
+            writeln!(
+                f,
+                "* {} ",
+                style("This token mint has a permanent delegate!").bold()
```

Comment:
```suggestion
                style("This token has a permanent delegate!").bold()
```

I know Jon commented on this line before, but I got confused because it reads like the account being displayed is a mint, and it isn't. Maybe go even simpler?

Reply:
yea i like this best actually